src/main/webapp/index.html

```diff
@@ -33,7 +33,21 @@ <h1>CodeU Starter Project</h1>
     <p>This is the CodeU starter project. Click the links above to login and visit your page.
     You can post messages on your page, and you can visit other user pages if you have
     their URL.</p>
-    <p>This is your code now! Feel free to make changes, and don't be afraid to get creative!
-    You could start by modifying this page to tell the world more about your team.</p>
+    <h2>Brandy Huang</h2>
+    <ul>
+      <li>intro:
+        So lucky to join this program and meet you guys!! I'm Sinyu Huang from Kaohsiung and now a sophomore studying electrical engineering in National Cheng Kong University.
+      I'm both interest in software and hardware. </li>
+      <li>interest:
+        <ol>
+          <li>Stock marketing: I was a member in stock studying club last year. We invite lecturers and discuss about financial big news happening in weeks. I have interest in studying options which is a financial product
+          . I don't operate stocks ,since I'm just a collage student without lots of money.haha~ However, always welcome everybody to discuss stocks with me!! </li>
+          <li>Cartoon Lover: Disney movies such as minions, Toy story, Wreck-It Ralph are my favorite. There are rare collage students go to theater watching cartoon, but I won't be absent in any disney movie. By the way, Aladdin will be released next week, but I have an important
+          midterm exam. That's miserable.</li>
+          <li>Love making friends: I'm the volunteer who helps international students in our school. I think that's a meaningful jobs which really helps people really needs. Since I'm a outgoing person and won't be nervous when talking strangers. The job really fits my personality.</li>
+        </ol>
+      </li>
+      <li>Also a coffee lovers!! Can't wait to meet you guys!!</li>
+    </ul>
```

Comment:
This is all good stuff, but not sure we want this in the index.html. I would move this into the aboutus.html and just put it in your section.

We can update index.html later to give an overview of our project (once we decide what we're going to build) and possibly a little about the team overall.